poller_servcheck.php

```diff
@@ -86,6 +91,10 @@
 	print "FATAL: You must install php-curl to use this Plugin" . PHP_EOL;
 }
 
+if (is_numeric($test_id) && $test_id > 0) {
+	$test_cond = ' AND id = ' . $test_id;
```

Comment:
The correct way to do this is to use `?` for the parameter value, and add test_id to the array of parameters to be used by the prepared query.